particles.f90

```diff
@@ -2192,7 +2196,8 @@ subroutine new_particle(idx,procidx)
 
       xv = ran2(iseed)*(xmax-xmin) + xmin
       yv = ran2(iseed)*(ymax-ymin) + ymin
-      zv = ran2(iseed)*zl
+      !zv = ran2(iseed)*zl
+      zv = zl
```

Comment:
Will this change any of the existing test case settings? Since this seems specific to open channel case